.github/workflows/ansible-test-plugins.yml

```diff
@@ -47,6 +48,7 @@ jobs:
           - stable-2.15
```

Comment:
```suggestion
```

Reply:
According to the [Ansible docs](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html), 2.15 will reach End of Life in 2 months. The likelihood of introducing an incompatible change for 2.15 is low. We could remove it now to avoid reopening a PR in 2 months.